transportation-data-publishing/open_data/pgrest_data_pub.py

```diff
@@ -97,15 +99,12 @@ def main():
     
     query_string = after_date_query(cfg_dataset["modified_date_field"], last_run_date)
 
-    records = pgrest.select(query_string)
-
+    records = pgrest.select(query_string, limit=limit)
+    
     if not records:
         return 0
 
-    date_fields = [
-        "traffic_report_status_date_time",
-        "published_date",
-    ]  # TODO: extract from API definition
+    date_fields = cfg_dataset.get('date_fields')
```

Comment:
what is `cfg_dataset`? I see it a bunch of places in this code but I don't know what it means.

Reply:
we import the config file as `cfg` which itself is a `dict` of configuration entries, one per `dataset`, so we assign the dataset-specific config to `cfg_dataset`. it's...goofy

Reply:
cool thanks for explaining. cfg = config = configuration. and each dataset has a configuration. Maybe dataset_config would be a better name?

> There are only two hard things in Computer Science: cache invalidation and naming things.